pkg/cmd/env/templates/rc.tmpl

```diff
@@ -2,9 +2,9 @@
 if [ -z "$GCTL_SESSION_ID" ] && [ -z "$TERM_SESSION_ID" ]; then
   export GCTL_SESSION_ID=$(uuidgen)
 fi
+alias {{.prefix}}=gardenctl
```

Comment:
we should have a consistent order across the shells
e.g. on zsh the alias is defined after the {{if not .noCompletion -}} block

Reply:
The completions work is very different in all shell. To have the same order in all shells I will move the completions behind the aliases. It would be strange to define completion for an alias that not has been defined yet even if it would work.

Reply:
✔️